recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "1.1.2" %}
+{% set version = "1.1.3" %}
 
 package:
   name: scikit-learn
   version: {{ version }}
 
 source:
   url: https://github.com/scikit-learn/scikit-learn/archive/{{ version }}.tar.gz
-  sha256: 92f3ad28149996caa9fd6ff5f849e7312d9fdedabe73760309a0e8b35b161735
+  sha256: b7c0a9fb8dfcefcfc7ef8fe02a064d194980251d57efaea972cb76005afb3c63
 
 build:
```

Comment:
Should we use `numpy` **1.22** for `py==310` and `win`?  See https://github.com/scipy/scipy/blob/de80faf9d3480b9dbb9b888568b64499e0e70c19/pyproject.toml#L42 and https://github.com/scikit-learn/scikit-learn/blob/ffc0f66676b4835eb1bdd3f3ecab025e9c1be9fe/pyproject.toml#L13-L17

Reply:
Their reason is specific to what the wheels on pypi contain. I don't see this affecting us.